client/src/main/java/org/dst/client/cmd/ArgsParseUtil.java

```diff
@@ -0,0 +1,49 @@
+package org.dst.client.cmd;
+
+import org.dst.common.exception.DstException;
+
+public class ArgsParseUtil {
+  private static final String defaultHost = "127.0.0.1";
+  private static final String defaultPort = "8082";
```

Comment:
Constant should be uppercase and separators("_") for splicing

Reply:
I have remove these two variables.